packages/shared/src/relay/getPreviewImageUrlsInlineDangerously.ts

```diff
@@ -0,0 +1,258 @@
+import { graphql, readInlineData } from 'relay-runtime';
```

Comment:
a newer and better version of our existing `getVideoOrImageUrlForNftPreview`
1. handles `SyncingMedia` types
2. no longer returns the incredibly confusing `video` type, which addresses one of our oldest TODOs and alleviates a lot of legacy headaches

<img width="718" alt="image" src="https://github.com/gallery-so/gallery/assets/12162433/ddba0ab3-32c3-4e6c-9c24-2a35f78ea040">